app/assets/images/payment_icons/luminor.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-luminor"><title id="pi-luminor">Luminor</title>
```

Comment:
Is it possible to add more spacing on the left and right side? It looks cramped

[![alt](https://screenshot.click/02-00-jqc7b-zhlwr.png)](https://screenshot.click/02-00-jqc7b-zhlwr.png)

Also, you would need to run the svg code through SVGO like in the [contribution](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#contributing) document.

Reply:
I ran the code through SVGO and added more spacing. As the SVG code is not very known for me I used https://vectorpaint.yaks.co.nz/ to add spacing. Hopefully it's good, please let me know :)